string eef_name
---
float64[] pose
